viafabric-mc120/src/main/java/com/viaversion/fabric/mc120/mixin/gui/client/MixinServerEntry.java

```diff
@@ -24,15 +24,7 @@ public class MixinServerEntry {
     @Final
     private ServerInfo server;
 
-    @Redirect(method = "render", at = @At(value = "INVOKE", ordinal = 0,
```

Comment:
Is it intentional, that this whole method is removed?

Reply:
Checking that rn.

Reply:
Yeah i had it removed because of `error: annotation type not applicable to this kind of declaration`.

Reply:
You should have updated it instead of deleting the whole code. Now the ping icon doesn't change texture when Via is translating the connection